### PR TITLE
Enable simd tests under wasm64. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by tlively** [comment]
LGTM assuming the tests pass.

**Review by tlively — APPROVED** [review]

**Comment by sbc100** [comment]
Actually it looks like clang is generate bad code in this config... I will open an llvm issue.

**Comment by sbc100** [comment]
opened https://github.com/llvm/llvm-project/issues/60963

**Comment by sbc100** [comment]
I reverted part of this PR since some tests passed and others didn't.

File: test/runner.py [hunk 1]
```diff
@@ -65,6 +65,8 @@
   'asan',
   'lsan',
   'ubsan',
+  'wasm64',
+  'wasm64_v8',
```
Comment (tlively) [inline test/runner.py:69]: Should this part also be reverted if the tests don't all pass?

**Review by tlively — APPROVED** [review]

File: test/runner.py (hunk 1 above)
Comment (sbc100) [reply test/runner.py:69]: No, all tests so pass, just some are disabled.  We run the full test suites in CI

**Pull request merged**